device/arista/x86_64-arista_7060x6_64pe_b/Arista-7060X6-64PE-B-P64/sai.profile

```diff
@@ -1,2 +1,3 @@
 SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/th5-a7060x6-64pe.config.bcm
-SAI_NUM_ECMP_MEMBERS=64
\ No newline at end of file
+SAI_NUM_ECMP_MEMBERS=64
+SAI_NHG_HIERARCHICAL_NEXTHOP=false
```

Comment:
@ccroy-arista when SAI_NHG_HIERARCHICAL_NEXTHOP is disabled, do we still need to set l3_ecmp_member_secondary_mem_size and l3_ecmp_member_first_lkup_mem_size to zero? Seems to be redundant, pls clarify.